modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/import.jsp

```diff
@@ -34,9 +34,11 @@ renderResponse.setTitle(LanguageUtil.get(resourceBundle, "import"));
 		<aui:input name="mvcPath" type="hidden" value="/admin/import.jsp" />
 		<aui:input name="parentKBFolderId" type="hidden" value="<%= String.valueOf(parentKBFolderId) %>" />
 
-		<liferay-ui:error exception="<%= KBArticleImportException.MustHaveACategory.class %>">
-			<%= LanguageUtil.format(request, "an-unexpected-error-occurred-while-importing-articles-x", LanguageUtil.get(request, "there-are-one-or-more-mandatory-vocabularies-assigned-to-the-knowledge-base-article")) %>
-		</liferay-ui:error>
+		<div class="mt-4">
```

Comment:
It is a bit strange to have this `div` envolving only this error. I tested it and looks necessary to have margin-top (however I would change it to `mt-3`, 16px is enough and looks better), so my suggestion is to move the rest of the message errors inside this div. In addition, for some reason it is not rendering correctly, look at the markup, the error is outside the `div`. Could you take a look please? 

![Screenshot 2023-02-01 at 11 33 43](https://user-images.githubusercontent.com/8373764/216027686-5d96145c-df70-455f-bfb7-420ca9793433.png)

Reply:
I've moved all the errors inside the div and now the markup looks well